src/textual/reactive.py

```diff
@@ -50,8 +42,6 @@ class Reactive(Generic[ReactiveType]):
         compute: Run compute methods when attribute is changed. Defaults to True.
     """
 
-    _reactives: TypeVar[dict[str, object]] = {}
```

Comment:
TypeVar is a mistake, but even more -- the whole thing is unused. I assume, again, remnants of old impl.

Reply:
Should be ClassVar, but `_reactives` is used. And this will give Mypy a bit more context.

Reply:
@willmcgugan if I understand it correct, `_reactives` is actually _not_ used. There's a bit of confusion here, and obviously I could be wrong, but it seems like `_reactives` is a class attribute of _Reactable_, not of `Reactive`. The class methods of `Reactives` traverse a Reatable's `obj`ect `._reactives` when running compute, watch, initialize_object, etc.

Also, a Reactable's `._reactives` is initialized in:
1. Declared in MessagePump
2. Initialized (and reset) in DOMNode
3. Initialized as a special case in App

The construct is confusing and I thought I'd take a stab as putting some order into this. So I am commenting on this PR but will create a new one soon with more changed lines of code but something that has a bit more order to it, I hope. Up to you.